src/com/tekion/cricket/Match.java

```diff
@@ -0,0 +1,267 @@
+package com.tekion.cricket;
+
+import java.util.Scanner;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Match {
+    private final Team team1;
+    private final Team team2;
+    enum Winner{
+        TEAM1,
+        TEAM2,
+        TIE,
+        STARTED,
+        DRAW,
```

Comment:
do we need comma in the end especially after `DRAW`

Reply:
No sir, I'll correct it.